vault-core/src/test/java/org/apache/jackrabbit/vault/packaging/integration/PackageInstallIT.java

```diff
@@ -381,12 +386,14 @@ public void testExternalHookFailsInInstalledPhase() throws RepositoryException,
      */
     @Test
     public void testNoProperties() throws RepositoryException, IOException, PackageException {
-        File tmpFile = File.createTempFile("vlttest", "zip");
-        IOUtils.copy(getStream("/test-packages/tmp_no_properties.zip"), FileUtils.openOutputStream(tmpFile));
-        JcrPackage pack = packMgr.upload(tmpFile, true, true, "testpackage", false);
```

Comment:
This should remove the tmpFile (due to the 2nd argument being true), for me that works, i.e. no tmpFiles left after IT execution